packages/shared/src/components/post/SquadPostContent.tsx

```diff
@@ -73,71 +73,72 @@ function SquadPostContent({
   const Content = ContentMap[finalType];
 
   return (
-    <>
-      {position === 'fixed' && (
-        <FixedPostNavigation
-          {...navigationProps}
-          isBannerVisible={isBannerVisible}
-          onReadArticle={onReadArticle}
-          className={className?.fixedNavigation}
-        />
```

Comment:
Diff looks crazy due to the removed unnecessary Fragment, but the only thing that changed here was this block passing it to as props.